ghost/collections/src/Collection.ts

```diff
@@ -69,7 +69,7 @@ export class Collection {
     }
 
     public async edit(data: Partial<Collection>, uniqueChecker: UniqueChecker) {
-        if (this.type === 'automatic' && this.slug !== 'index' && (data.filter === null || data.filter === '')) {
```

Comment:
Wondering if we should instead have:
```
if (this.slug === 'latest' || this.slug === 'featured') {
  throw new ValidationError('Cannot edit internal collections');
}
```

Reply:
We might 🤔 As for now we don't have a requirement for the built-in collections to be "immutable". Maybe something we'd add eventually if that comes up as a feature. It looks like an obvious one to have but don't think we need it just yet.